docs/specification/appendix.md

```diff
@@ -107,6 +107,28 @@ The IDL definition of the type specified by `name` along with all dependent type
 - a line containing exactly 80 `=` characters, then
 - A line containing only `IDL: <package resource name>` for that definition. The space between `IDL:` and the package resource name is mandatory. The package resource name does not include a file extension.
 
+### omgidl
+
+The `omgidl` schema encoding stores [OMG IDL](https://www.omg.org/spec/IDL/4.2/About-IDL) definitions for (X)CDR-encoded messages.
+
+- `name`: The globally-scoped name of the message type, eg. `my_company::my_module::MyType`
```

Comment:
Is the `my_company` part in the idl schemas as a module?

Reply:
yes - not sure how to make that more obvious.

Reply:
Should this specify that this globally-scoped name should be in the `data` exactly and will be used as the root schema when decoding the message? Or was there another way we wanted to specify that?

Reply:
Maybe `my_company_global_module::my_specific_module::MyType`...?